packages/design-system/src/scripts/components/DateField/DateField.jsx

```diff
@@ -114,6 +114,7 @@ export class DateField extends React.PureComponent {
             name={this.props.monthName}
             value={this.props.monthValue}
             aria-describedby={labelId}
+            autoComplete={this.props.autoComplete && 'bday-month'}
```

Comment:
I'm thinking using something more generic for the prop names would serve more use cases in the future.
Something like `dayAutocomplete`, `monthAuthocomplete`, `yearAutocomplete`.

Reply:
Date field autocomplete is currently available for birthdate and credit card month & year only and the change as of right now only applies for birth day, month and year. 

To be more generic and caters for autocomplete of birth date, credit card date and future cases, your suggestion makes sense. So if props are defined for `dayAutocomplete`, `monthAutocomplete` and `yearAutocomplete`, then when using `Date field`, these props can be set to autocomplete for any token depending on use cases. 

Example: 
- `yearAutocomplete='bday-year' ` to autocomplete for birth year
 - `yearAutocomplete='cc-exp-year'` to autocomplete for credit card year.

Reply:
yeah, I touched on this with Sawyer in the [original issue](https://github.com/CMSgov/design-system/issues/565#issuecomment-574250780). I think since the only autocomplete case that we can currently anticipate is for birthdays (credit cards don't need the full month/date/year), it's easier on the user to just have a single `autoComplete` prop. I can't think of a case where a user would want to have autocomplete only on the year for example, so I dont think we need to provide that level of granularity.

Reply:
Yes, we'll have simplicity with just one autcomplete prop